### PR TITLE
fix: Move docker command under utils/scripts directory

### LINKED ISSUE #589: The keyword should not be tied to docker commands
Currently the default deployment of EdgeX is docker, but it should not be limited to docker.
Some keywords are depends on the `docker logs` or `docker exec` commands, and they should be enhanced to allow non-docker environment.  The approach should work like start and restart services.
https://github.com/edgexfoundry/edgex-taf/blob/5d08e5869bd0fc16a8a9c08b7197e0a68f4bdbf5/TAF/testCaseModules/keywords/common/commonKeywords.robot#L254
https://github.com/edgexfoundry/edgex-taf/blob/71fd72e5bb3027d24703a67c6a37c126e5661488/TAF/testCaseModules/keywords/setup/edgex.py#L104
https://github.com/edgexfoundry/edgex-taf/blob/4f2f1212cb34d94110257ce33c5f63a67df7c113/TAF/testScenarios/integrationTest/UC_mqtt_message_bus/device_virtual_config.robot#L103
https://github.com/edgexfoundry/edgex-taf/blob/0528d3ff5131d7c2ecdc45d4dccb1536ab65467e/TAF/testScenarios/functionalTest/V2-API/app-service/secrets/POST.robot#L50

### PR DESCRIPTION
**Pull request by cherrycl**
Fix #589 
Signed-off-by: Cherry Wang <cherry@iotechsys.com>

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Comment by cherrycl** [comment]
Hi @siggiskulason, would you please have a look if this change effect the snap script? If yes, you may need to update the snap script after the PR merged.

File: TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot [hunk 1]
```diff
@@ -100,17 +100,19 @@ Start HTTP Server And Received Exported Data Contains ${keyword}
                              ...       ELSE  Get Line Count  ${http_server_received}
     Should Be Equal As Integers  1  ${http_received_length}
 
-Found Retry Log ${number} Times In app-http-export Logs From ${timestamp}
-    ${logs}  Get Service Logs Since Timestamp  app-http-export  ${timestamp}
-    Log  ${logs}
-    ${retry_lines}  Get Lines Containing String  ${logs}.encode()  1 stored data items found for retrying
+Found Retry Log ${number} Times In ${service_name} Logs From ${timestamp}
+    ${timestamp}  Evaluate  int(${timestamp})
```
Comment (jinlinGuan) [inline TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot:104]: This line can be removed.

File: TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot [hunk 2]
```diff
@@ -100,17 +100,19 @@ Start HTTP Server And Received Exported Data Contains ${keyword}
...
+    ${retry_lines}  Get Lines Containing String  ${logs.stdout}.encode()  1 stored data items found for retrying
     ${retry_times}  Get Line Count  ${retry_lines}
     Should Be Equal As Integers  ${retry_times}  ${number}
 
-Found Remove Log In app-http-export Logs From ${timestamp}
-    ${logs}  Get Service Logs Since Timestamp  app-http-export  ${timestamp}
-    Log  ${logs}
-    ${retry_lines}  Get Lines Containing String  ${logs}.encode()  1 stored data items will be removed post retry
+Found Remove Log In ${service_name} Logs From ${timestamp}
+    ${timestamp}  Evaluate  int(${timestamp})
```
Comment (jinlinGuan) [inline TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot:112]: This line can be removed.

**Review by jinlinGuan — CHANGES_REQUESTED** [review]

File: TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot (hunk 1 above)
Comment (cherrycl) [reply TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot]: Removed.

File: TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot (hunk 2 above)
Comment (cherrycl) [reply TAF/testScenarios/integrationTest/UC_end_to_end/export_store_forward.robot]: Removed.

**Review by jinlinGuan — APPROVED** [review]
LGTM

**Pull request merged**